docs/content/getting-started/setup-openfga.mdx

```diff
@@ -53,7 +53,31 @@ All possible configurations and their default values are defined in [config-sche
 You can choose between two storage options:
 
 - In-memory storage. This option is not persistent, i.e. if you restart the server, the data will be lost.
-- Postgres database. This option requires either a local Postgres installation, or a Docker container that runs Postgres. For an example of a Docker setup that includes a Postgres database, see [OpenFGA's Docker compose file](https://github.com/openfga/openfga/blob/main/docker-compose.yaml).
+- Postgres database. This option requires either a local Postgres installation, or a Docker container that runs Postgres.
+
```

Comment:
### With docker compose

1. Copy the following snippet and paste it into a file. Save it as `docker-compose.yaml`
```yaml
version: '3.8'

networks:
  openfga:

services:
  postgres:
    image: postgres:14
    networks:
      - openfga
    environment:
      - POSTGRES_USER=postgres
      - POSTGRES_PASSWORD=password
      - POSTGRES_DB=openfga
    healthcheck:
      test: [ "CMD-SHELL", "pg_isready -U postgres" ]
      interval: 5s
      timeout: 5s
      retries: 5

  openfga:
    image: openfga/openfga
    environment:
      - OPENFGA_DATASTORE_ENGINE=postgres
      - OPENFGA_DATASTORE_CONNECTION_URI=postgres://postgres:password@postgres:5432/openfga
      - OPENFGA_LOG_FORMAT=json
    command: run
    depends_on:
      - postgres
    networks:
      - openfga
    ports:
      - "8080:8080"
      - "8081:8081"
      - "3000:3000"
```

2. Run the postgres container in the background

```shell
docker-compose up -d postgres
```

3. Run the necessary OpenFGA migrations

```shell
docker-compose run openfga migrate
```

4. Run the OpenFGA server
```shell
docker-compose up openfga
```

5. Verify that you can reach the openfga server
```shell
curl http://localhost:8080/stores
```

You should see the following output:
```json
{"stores":[], "continuation_token":""}
```

Reply:
I don't love that a person trying to do this will have to write a new file. As a user, I would find it easier to just copy and paste commands in the terminal. WDYT?

Reply:
I agree with @craigpastro actually. I like it the way it is, because the commands are fully self-contained (no extra file creation/copying necessary).